bps/templates/bps_defects.yaml

```diff
@@ -2,10 +2,11 @@ pipelineYaml: "${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpDefects.yaml"
 
 project: "${TICKET}"
 campaign: "${TICKET}"
+submitPath: "$SCRATCH/submit/{outputRun}"
 
-biasBootstrapRun: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/biasBootstrapGen.$RERUN/run${BOOTSTRAP_RUN_NUMBER}"
-darkBootstrapRun: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/darkBootstrapGen.$RERUN/run${BOOTSTRAP_RUN_NUMBER}"
-flatBootstrapRun: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/flatBootstrapGen.$RERUN/run${BOOTSTRAP_RUN_NUMBER}"
+biasBootstrapRun: "${USER_CALIB_PREFIX}$INSTRUMENT/calib/$TICKET/$TAG/biasBootstrapGen.${RERUN}/run${BOOTSTRAP_RUN_NUMBER}"
```

Comment:
Do any of the `$RERUN` variables elsewhere need braces as well?  I see above in the `bps_darkBootstrap.yaml` file that we have `$RERUN`.  Similarly in the `bps_flatBootstrap.yaml`.

Reply:
I think that the braces are optional, but I meant to standardize on using it (like the quotes) in which case `$TICKET/$TAG` should be updated as well (here and below)